NYPLCardCreator/Classes/AddressViewController.swift

```diff
@@ -3,16 +3,6 @@ import UIKit
 /// This class is used for allowing the user to enter an address.
 final class AddressViewController: FormTableViewController {
   
-  private static let regions: [String] = {
```

Comment:
If we're definitely removing the picker view, be sure to also remove "regions.json" from the project so it's not hanging around doing nothing.